### PR TITLE
Log file updates

### PR DESCRIPTION
**Pull request by wtgee**
Updates to log files.

Replaces #160

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -22,22 +26,80 @@ playground rules and follow them during all your contributions.
...
+ [Coding in PANOPTES](https://github.com/panoptes/POCS/wiki/Coding-in-PANOPTES) page.
 
 
 # Code Formatting
 
 - All Python should use [PEP 8 Standards](https://www.python.org/dev/peps/pep-0008/)
-   - Use a tool such as [yapf](https://github.com/google/yapf) to format your
-     files; we'd rather spend time developing PANOPTES and not arguing about
-     style.
+   - Line length is set at 120 characters instead of 80
```
Comment (jamessynge) [inline CONTRIBUTING.md]: 100, not 120, right?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -22,22 +26,80 @@ playground rules and follow them during all your contributions.
...
+   - It is recommended to have your editor auto-format code whenever you save a file rather than 
+   attempt to go back and change an entire file all at once. 
 - Do not leave in commented-out code or unnecessary whitespace.
-- Variable/function/class and file names should be meaningful and descriptive
-- File names should be lower case and underscored, not contain spaces. For
-  example, `my_file.py` instead of `My File.py`
-- Define any project specific terminology or abbreviations you use in the file you use them
+- Variable/function/class and file names should be meaningful and descriptive.
+- File names should be underscored, not contain spaces ex. my_file.py.
```
Comment (jamessynge) [inline CONTRIBUTING.md]: Please keep the old file names description (lower case, etc.).

File: pocs/utils/logger.py [hunk 3]
```diff
@@ -23,23 +50,41 @@ def get_root_logger(profile='panoptes', log_config=None):
...
+            finally:
+                os.symlink(full_log_fname, log_symlink)
 
     # Configure the logger
     logging.config.dictConfig(log_config)
 
     # Get the logger and set as attribute to class
     logger = logging.getLogger(profile)
 
+    # Don't want log messages from state machine library
```
Comment (jamessynge) [inline pocs/utils/logger.py]: I appreciate the comment as I had no idea what this was about. I wonder if this is the kind of thing that could be in conf_files/log.yaml. What do you think?

File: CONTRIBUTING.md (hunk 1 above)
Comment (wtgee) [reply CONTRIBUTING.md]: Arg...this and the comment below are the same as what you saw happening with the serial testing PR where it is pulling in the old items.

**Comment by wtgee** [comment]
Closing in favor of #165, which is the same thing (with corrections from above) but without the corrupted history.

**Pull request closed without merging**